dev-tools/mage/pkgtypes.go

```diff
@@ -646,6 +648,17 @@ func runFPM(spec PackageSpec, packageType PackageType) error {
 	if spec.URL != "" {
 		args = append(args, "--url", spec.URL)
 	}
+	if spec.User != "" {
+		args = append(args, "--rpm-user", spec.User)
```

Comment:
Great catch, I'll rework this.

Reply:
Switched to per-file ownership instead of package level in 12f9766469db80cc809ac4d89e24d1ffa0e7bc35.  Unfortunately only rpm has a per-file override in `--rpm-attr`.  Since `--deb-use-file-permissions` is on, it might be possible to get this working in deb too - I'm not sure it's important as I haven't found a deb equivalent of `rpm setperms` either.  For debs a post-install permissions setting for config file, `/var/{lib,log}/beat` will get accomplish what apm-server needs cleanly enough - we'll do that in rpms too to handle upgrades.